client/src/components/graph/setupLasso.js

```diff
@@ -7,9 +7,15 @@ const Lasso = () => {
   const dispatch = d3.dispatch("start", "end", "cancel");
 
   const lasso = (svg) => {
-    let lassoPolygon;
-    let lassoPath;
-    let closePath;
+    let lassoPolygon = null;
+    let lassoPath = null;
+    let closePath = null;
+
+    const resetState = () => {
```

Comment:
rather than defining a new function, why not just use the existing `reset()` function?